sdcm/cluster.py

```diff
@@ -3193,6 +3193,21 @@ def __init__(self, cluster_uuid=None, cluster_prefix='cluster', node_prefix='nod
             self.run_node_benchmarks()
         self.coredumps = {}
         super().__init__()
+        self.keyspace_info = []
+
+    def set_keyspace_info(self, keyspace_name: str, db_node: BaseNode):
```

Comment:
my proposal is to remove this method, include setting `keyspace_info` while running `get_keyspace_info`

also, `self.keyspace_info` could be a dict containing keyspace names as keys and `ReplicationStrategy` (with additional attrs as mentioned in other comment) as values.

Reply:
changed accordingly to comments